### ISSUE TITLE
Running command from phone not working

### ISSUE THREAD

**Issue by daudix-UFO** [andyholmes/valent#450]
### Current Behavior

Running command from phone doesn't work, while this does work with GSConnect

### Expected Behavior

A command to run from phone from app, homescreen widget or home controls

### Desktop

GNOME Shell

### Other Desktop

GNOME

### Operating System

Fedora Silverblue 38

### Installed from

Nightly Flatpak

### Version

1.0.0.alpha

### Devices

KDE Connect (Android), Valent (Desktop Linux)

### Plugins

Commands

### Logs

```shell
(flatpak-spawn:31): GLib-GIO-DEBUG: 01:49:07.001: Using cross-namespace EXTERNAL authentication (this will deadlock if server is GDBus < 2.73.3)
** (flatpak-spawn:31): DEBUG: 01:49:07.005: child_pid: 30096
** (flatpak-spawn:31): DEBUG: 01:49:07.005: child exited 30096: 0
** (flatpak-spawn:31): DEBUG: 01:49:07.005: child exit code 30096: 0
```

[:paperclip: Full log](https://github.com/andyholmes/valent/files/11278133/valent-log.txt)

### Screenshots

[Kooha-2023-04-20-02-03-24.webm](https://user-images.githubusercontent.com/77155297/233218154-3be36e02-4172-4906-88a9-739a494ad61f.webm)

**Comment by andyholmes** [comment]
Thanks for reporting, I think this should be fixed in #452.

**Comment by daudix-UFO** [comment]
Thank you very much!
